frontend/components/UnderDev.tsx

```diff
@@ -6,11 +6,11 @@ const UnderDev = () => {
       <Head>
         <title>Under development</title>
       </Head>
-      <div className='m-auto grid gap-10 place-self-center text-center dark:text-white lg:w-[60rem]'>
-        <h1 className='text-3xl font-bold md:text-5xl lg:text-7xl'>
+      <div className='grid gap-10 m-auto text-center place-self-center text-secondary dark:text-secondary-dark'>
```

Comment:
As mentioned before, you might want to use the accent color for this and secondary for the `<p>` below